schema/schema.go

```diff
@@ -136,41 +134,14 @@ type Factory struct {
 	// schema subspace.
 	Schema jsoniter.RawMessage
 	// CollectionType is the type of the collection. Only two types of collections are supported "messages" and "documents"
-	CollectionType  CollectionType
-	IndexingVersion string
-	Version         uint32
+	CollectionType CollectionType
+	Version        uint32
 }
 
 func (f *Factory) SecondaryIndexes() []*Index {
 	return f.Indexes.All
 }
 
-func RemoveIndexingVersion(schema jsoniter.RawMessage) jsoniter.RawMessage {
```

Comment:
Can you cleanup `IndexingSchemaVersionKey` and `DefaultIndexingSchemaVersion` as well?